controllers/useraccount/useraccount_controller_test.go

```diff
@@ -2000,46 +1744,6 @@ func terminating(msg string) toolchainv1alpha1.Condition {
 	}
 }
 
-func cheRoute(tls bool) *routev1.Route { //nolint: unparam
-	return testRoute("codeready", "codeready-workspaces-operator", "codeready-codeready-workspaces-operator", tls)
-}
-
-func keycloackRoute(tls bool) *routev1.Route { //nolint: unparam
-	return testRoute("keycloak", "codeready-workspaces-operator", "keycloak-codeready-workspaces-operator", tls)
-}
-
-func testRoute(name, namespace, serviceName string, tls bool) *routev1.Route { //nolint: unparam
-	r := &routev1.Route{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
-		},
-		Spec: routev1.RouteSpec{
-			Host: fmt.Sprintf("%s.%s", serviceName, test.MemberClusterName),
-			Path: "",
-		},
-	}
-	if tls {
-		r.Spec.TLS = &routev1.TLSConfig{
-			Termination: "edge",
-		}
-	}
-	return r
-}
-
-func newSecretWithCheAdminCreds() *corev1.Secret {
-	return &corev1.Secret{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "test-secret",
-			Namespace: test.MemberOperatorNs,
-		},
-		Data: map[string][]byte{
-			"che.admin.username": []byte("test-che-user"),
-			"che.admin.password": []byte("test-che-password"),
-		},
-	}
-}
-
 func gockTokenSuccess(calls *int) {
 	gock.New(testKeycloakURL).
 		Post("auth/realms/codeready/protocol/openid-connect/token").
```

Comment:
I guess that we can delete this gock mock and also the other ones in this file.

Reply:
Done. 795fc707ddaa208c398e6e70b8693f26faf6d90b